js/site.js

```diff
@@ -67,7 +68,7 @@ function generatePositionValues(pre_pos) {
 }
 
 // Creates an svg containing n circles
-function cloudSVG(n, precip) {
+function cloudSVG(n, p) {
```

Comment:
I personally tend to try and use somewhat descriptive variable names. Helps with debugging and comprehending the code later.

Reply:
Noted, will adjust